.gitignore

```diff
@@ -33,3 +33,5 @@ temp.dmi
 /maps/RandomZLevels/backup
 /tools/midi2piano.zip
 /maps/yw/backup
+
+icons/mecha/mecha_equipment_ch.dmi

```

Comment:
Mlem? Might want to remove this from being ignored.

Reply:
heck, I ignored the wrong file.